internal/pkg/fakeroot/fakeroot.go

```diff
@@ -22,16 +22,16 @@ func GetIDRange(base uint64, allowedUsers []string) (*specs.LinuxIDMapping, erro
 	}
 	if base%65536 != 0 {
 		return nil, fmt.Errorf("fakeroot base id is not a multiple of 65536")
-	} else if base < 65536 || base > 4294901760 {
-		return nil, fmt.Errorf("fakeroot base id is not set between 65536 and 4294901760")
+	} else if base < 131072 || base > 4294901760 {
```

Comment:
please give these magic numbers a name so that it's easier to understand what's happening when they get changed.